cmd/argo/commands/archive/get.go

```diff
@@ -51,7 +51,11 @@ func NewGetCommand() *cobra.Command {
 				fmt.Printf(fmtStr, "Namespace:", wf.ObjectMeta.Namespace)
 				serviceAccount := wf.Spec.ServiceAccountName
 				if serviceAccount == "" {
-					serviceAccount = "default"
+					// if serviceAccountName was not specified in a submitted Workflow, we will
+					// use the serviceAccountName provided in Workflow Defaults (if any). If that
+					// also isn't set, we will use the 'default' ServiceAccount in the namespace
+					// the workflow will run in.
+					serviceAccount = "unset (using default ServiceAccount instead)"
```

Comment:
```suggestion
					serviceAccount = "unset (will run with the default ServiceAccount)"
```

Reads just _slightly_ clearer to me, but feel free to disregard. Might just be personal preference.

Reply:
Yep, that's clearer indeed. Changed it 😊